generator/rockset.go

```diff
@@ -54,11 +54,41 @@ func (r *Rockset) SendDocument(docs []any) error {
 	return nil
 }
 
+func (r *Rockset) SendPatch(docs []interface{}) error {
+	numDocs := len(docs)
+	rcollection := strings.Split(r.collectionPath, ".") // this is already validated two have two components
```

Comment:
```suggestion
	rcollection := strings.Split(r.collectionPath, ".") // this is already validated to have two components
```

sorry this is probably my typo 🤦